frontend/src/metabase/public/components/EmbedFrame/LogoBadge.styled.tsx

```diff
@@ -0,0 +1,26 @@
+import styled from "@emotion/styled";
+import ExternalLink from "metabase/core/components/ExternalLink";
+import { color } from "metabase/lib/colors";
+import { breakpointMinMedium } from "metabase/styled-components/theme";
+
+export const MetabaseLink = styled(ExternalLink)`
+  display: flex;
+  align-items: center;
+
+  font-size: 0.85rem;
+  font-weight: bold;
+  text-decoration: none;
+`;
+
+export const Message = styled.span`
```

Comment:
Worth noting that we're removing nested `span`s and only producing a single `span` instead. I don't see any problem with that.

Also property for `.text-small` is missing, unless you handle this another way.
https://github.com/metabase/metabase/blob/62f05deaad7523eeb22f883f1bc5ee528802c092/frontend/src/metabase/css/core/text.css#L163-L165

Can you make sure this still renders the same font size?

Reply:
Yeah, please check out [this comment](https://github.com/metabase/metabase/pull/27746#discussion_r1082621727) above